lib/config/env.dart

```diff
@@ -25,4 +25,6 @@ abstract class Env {
   static final String wiredashId = _Env.wiredashId;
   @EnviedField()
   static final String wiredashSecret = _Env.wiredashSecret;
+  @EnviedField()
+  static final String sksUrl = _Env.sksUrl;
```

Comment:
if you want the build to pass you have to add this to example.env too (even and empty string)

Reply:
ah ok